ci/pipeline.yml

```diff
@@ -444,6 +469,12 @@ resources:
   -  #@ deps_version_resource()
   -  #@ bundled_deps_resource()
   -  #@ slack_resource()
+  - name: galoy-mobile-repo
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/GaloyMoney/galoy-mobile
+      branch: main
```

Comment:
@bodymindarts This is probably supposed to be injected in. Do we have a common shared repo for github resources?